.CI/common.groovy

```diff
@@ -110,21 +212,34 @@ void buildGUI(stash) {
   sh 'CONFIG=`./config.status --config` && ./configure `eval $CONFIG`'
   sh "touch omc.skip omc-diff.skip ReferenceFiles.skip omsimulator.skip && ${makeCommand()} -q omc omc-diff ReferenceFiles omsimulator" // Pretend we already built omc since we already did so
   sh "${makeCommand()} -j${numPhysicalCPU()} --output-sync" // Builds the GUI files
+
+  }
 }
 
 void generateTemplates() {
+  if isWindows() {
+  // do nothing
+  } else {
   patchConfigStatus()
   // Runs Susan again, for bootstrapping tests, etc
   sh "${makeCommand()} -C OMCompiler/Compiler/Template/ -f Makefile.in OMC=\$PWD/build/bin/omc"
   sh 'cd OMCompiler && ./config.status'
   sh './config.status'
+  }
 }
 
 def getVersion() {
+  if isWindows() {
+  return (sh (script: 'set OMDEV=C:\OMDev && set MSYSTEM=MINGW64 && %OMDEV%\tools\msys\usr\bin\sh --login -i -c "build/bin/omc --version | grep -o \"v[0-9]\\+[.][0-9]\\+[.][0-9]\\+[^ ]*\""', returnStdout: true)).replaceAll("\\s","")
+  } else {
   return (sh (script: 'build/bin/omc --version | grep -o "v[0-9]\\+[.][0-9]\\+[.][0-9]\\+[^ ]*"', returnStdout: true)).replaceAll("\\s","")
+  }
 }
 
 void compliance() {
+  if isWindows() {
```

Comment:
You could skip this function. We don't need to run the compliance suite on Windows, do we?

Reply:
i don't know, maybe yes